hackathon_site/registration/models.py

```diff
@@ -99,19 +106,63 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+    linkedin = models.URLField(
+        max_length=200, help_text="LinkedIn Profile", null=True, blank=True
+    )
+    github = models.URLField(
```

Comment:
I checked to see if we have a way of differentiating between required and optional fields but we don't (probably something we should look into) so could you put `(Optional)` in the help_text for all the profile links